test/cereal_test.exs

```diff
@@ -2,7 +2,50 @@ defmodule CerealTest do
   use ExUnit.Case
   doctest Cereal
 
-  test "the truth" do
-    assert 1 + 1 == 2
+  defmodule UserSerializer do
+    use Cereal.Serializer
+    attributes [:name, :property_that_needs_context]
+
+    def property_that_needs_context(%{_context: context}, _) do
```

Comment:
It feels quite odd to me that we are attaching this ephemeral attribute to the data-to-be-serialized. At the very least, it's a munging of concerns b/t the library (Cereal) and the client (Frame.io API, or other). Maybe instead we assign it to the `conn` that's passed in as the second argument of attribute functions?